### PR TITLE
man: document missing KillSignal= .nspawn option

### PR DESCRIPTION
**Pull request by petrosagg**
This option works if specified but was missing from the manpage.

### CODE REVIEW

File: man/systemd.nspawn.xml [hunk 1]
```diff
@@ -224,6 +224,19 @@
...
+        container's PID 1 when nspawn itself receives SIGTERM, in
+        order to trigger an orderly shutdown of the
+        container. Defaults to SIGRTMIN+3 if <option>Boot=</option>
+        is used (on systemd-compatible init systems SIGRTMIN+3
+        triggers an orderly shutdown). Takes a signal name like
+        <literal>SIGHUP</literal>, <literal>SIGTERM</literal> or
+        similar as argument.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
```
Comment (zonque) [inline man/systemd.nspawn.xml:238]: Please replace that last sentence with `For a list of valid signals, see signal(7).`

**Comment by zonque** [comment]
lgtm otherwise!

File: man/systemd.nspawn.xml (hunk 1 above)
Comment (zonque) [inline man/systemd.nspawn.xml:238]: And by that, I mean a real reference to that other man page. See `man/systemd.kill.xml`.

Comment (petrosagg) [inline man/systemd.nspawn.xml:238]: FIxed. Also changed `systemd-nspawn.xml` to match this

**Comment by keszybz** [comment]
Samaphore build failed with ICE, great.

**Pull request merged**